src/main/scala/io/sdkman/changelogs/java/AdoptOpenJdkMigrations.scala

```diff
@@ -23,4 +23,27 @@ class AdoptOpenJdkMigrations {
       removeVersion("java", "13.0.0.hs-adpt", platform)
     }
   }
+
+  @ChangeSet(order = "0002", id = "0002-add_adoptopenjdk-hs_11_0_6", author = "poad")
+  def migrate0002(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "11.0.6.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.6+10/OpenJDK11U-jdk_x64_linux_hotspot_11.0.6_10.tar.gz", Linux64, Some(AdoptOpenJDK)),
+      Version("java", "11.0.6.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.6+10/OpenJDK11U-jdk_x64_mac_hotspot_11.0.6_10.tar.gz", MacOSX, Some(AdoptOpenJDK)),
+      Version("java", "11.0.6.hs-adpt", "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.6+10/OpenJDK11U-jdk_x64_windows_hotspot_11.0.6_10.zip", Windows, Some(AdoptOpenJDK)))
+      .validate()
+      .insert()
```

Comment:
11.0.6.hs-adpt should be set as the new default version as part of the migration.

Reply:
@marc0der Thank you!.
Added the set default.
Please check again.